### PR TITLE
Improve setup information

### PR DESCRIPTION
**Pull request by drapergeek**
- Add ruby version file
- Create a setup script

Please do not merge this PR. If someone can sign off and make sure they approve I'll clean up the commits and merge once it is reviewed. Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,18 @@
 
 ## Setup
 
+This project uses a `.ruby_version` to specify what version of ruby to use. You
+can use a number of ruby version managers to get this. A list of common options
+if provided below.
```
Comment (jtslear) [inline README.md]: should be `is provided below.`

**Comment by jtslear** [comment]
Aside from a very minor comment, this PR looks good.

**Comment by jtslear** [comment]
resolves #3

**Comment by drapergeek** [comment]
Thanks @jtslear

**Pull request merged**